test/CMakeLists.txt

```diff
@@ -1,2 +1,15 @@
 # Use the example executable as a test
-add_test(example ${CMAKE_BINARY_DIR}/example_dir/example)
\ No newline at end of file
+add_test(example ${CMAKE_BINARY_DIR}/example_dir/example)
+
+# Require Boost testing framework for real tests
```

Comment:
What does "real" mean in this sentence?

Reply:
Up to this point, we had the example as the only test in this repo, and that does not look like a real test, nothing is checked.

Reply:
ahaha.